src/enums/DatatypeEnum.php

```diff
@@ -49,22 +54,20 @@ class DatatypeEnum
      *
      * @return bool|string
      */
-    public static function getValidTypeFromSample($value){
+    public static function getValidTypeFromSample($value) {
```

Comment:
When writing this function I never was sure if this is the right class to declare it. I think it's not fully correct.